variables.tf

```diff
@@ -278,6 +278,12 @@ variable "monitoring_sns_protocol" {
   default     = "email"
 }
 
+variable "monitoring_enable_detailed_monitoring" {
+  description = "If true, the launched EC2 instance will have detailed monitoring enabled"
```

Comment:
If this is only for ec2, maybe rename the variable to `monitoring_enable_detailed_instance_monitoring`